### PR TITLE
fix small bug in find_eb_script + various minor issues with tests (related to environment in which tests are run)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: test/framework/options.py [hunk 1]
```diff
@@ -2248,7 +2270,11 @@ def test_include_module_naming_schemes(self):
         args = [
             '--avail-module-naming-schemes',
         ]
-        logtxt, _ = run_cmd("cd %s; eb %s" % (self.test_prefix, ' '.join(args)), simple=False)
+
+        env_cmd = "export PATH='%s'; export PYTHONPATH='%s'; " % (path, pythonpath)
```
Comment (akesandgren) [inline test/framework/options.py]: Don't really need the "; export" part here you can export both with the same export

File: test/framework/options.py [hunk 2]
```diff
@@ -2326,7 +2366,10 @@ def test_include_toolchains(self):
         args = [
             '--list-toolchains',
         ]
-        logtxt, _ = run_cmd("cd %s; eb %s" % (self.test_prefix, ' '.join(args)), simple=False)
+        env_cmd = "export PATH='%s'; export PYTHONPATH='%s'; " % (path, pythonpath)
```
Comment (akesandgren) [inline test/framework/options.py]: Same here

File: test/framework/options.py [hunk 3]
```diff
@@ -2271,7 +2271,7 @@ def test_include_module_naming_schemes(self):
             '--avail-module-naming-schemes',
         ]
 
-        env_cmd = "export PATH='%s'; export PYTHONPATH='%s'; " % (path, pythonpath)
+        env_cmd = "export PATH='%s' export PYTHONPATH='%s'; " % (path, pythonpath)
```
Comment (akesandgren) [inline test/framework/options.py]: No, env_cmd = "export PATH='%s' PYTHONPATH='%s'; " % (path, pythonpath)

Comment (boegel) [reply test/framework/options.py]: needs more coffee...

Comment (boegel) [reply test/framework/options.py]: fixed in 776993e6e with force push

**Comment by akesandgren** [comment]
Going in, thanks @boegel!

**Pull request merged**